plugins/poll/assets/javascripts/discourse/components/poll-info.gjs

```diff
@@ -0,0 +1,210 @@
+import Component from "@glimmer/component";
+import { inject as service } from "@ember/service";
+import { htmlSafe } from "@ember/template";
+import { relativeAge } from "discourse/lib/formatter";
+import dIcon from "discourse-common/helpers/d-icon";
+import I18n from "I18n";
+
+export default class PollInfoComponent extends Component {
+  @service currentUser;
+
+  get multipleHelpText() {
+    const min = this.args.min;
+    const max = this.args.max;
+    const optionsCount = this.args.options.length;
+
+    if (max > 0) {
+      if (min === max) {
+        if (min > 1) {
+          return htmlSafe(
+            I18n.t("poll.multiple.help.x_options", { count: min })
+          );
+        }
+      } else if (min > 1) {
+        if (max < optionsCount) {
+          return htmlSafe(
+            I18n.t("poll.multiple.help.between_min_and_max_options", {
+              min,
+              max,
+            })
+          );
+        } else {
+          return htmlSafe(
+            I18n.t("poll.multiple.help.at_least_min_options", {
+              count: min,
+            })
+          );
+        }
+      } else if (max <= optionsCount) {
+        return htmlSafe(
+          I18n.t("poll.multiple.help.up_to_max_options", { count: max })
+        );
+      }
+    }
```

Comment:
```suggestion
if (max > 0) {
    if (min === max && min > 1) {
      return htmlSafe(
        I18n.t("poll.multiple.help.x_options", { count: min })
      );
    }

    if (min > 1) {
      if (max < optionsCount) {
        return htmlSafe(
          I18n.t("poll.multiple.help.between_min_and_max_options", { min, max })
        );
      } 
      
      return htmlSafe(
        I18n.t("poll.multiple.help.at_least_min_options", { count: min })
      );
    }

    if (max <= optionsCount) {
      return htmlSafe(
        I18n.t("poll.multiple.help.up_to_max_options", { count: max })
      );
    }
  }
```

I think this is equivalent and simpler, but please double check

Reply:
@jjaffeux thanks, neater!  Note I didn't author this, I just carried it across.